### PR TITLE
Converting Monitoring project to .netStandard, while the support for …

### PR DESCRIPTION
**Pull request by AvitalOfstein**
….net451 stays as well (backwards compatability).

Tets project will be in .net461 framework.

### CODE REVIEW

**Review by DannyVarod — APPROVED** [review]

File: Monitoring/Monitoring.csproj [hunk 1]
```diff
@@ -1,87 +1,20 @@
...
-    </Compile>
-    <Compile Include="LogExceptionsAttribute.cs" />
-    <Compile Include="LogAttribute.cs" />
-    <Compile Include="LogLevelExtensions.cs" />
-    <Compile Include="LogLevelValue.cs" />
-    <Compile Include="MonitorStatistics.cs" />
-    <Compile Include="MonitorAttribute.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="SpinMonitor.cs" />
+    <Folder Include="Properties\" />
```
Comment (linoyisr) [inline Monitoring/Monitoring.csproj]: <Folder Include="Properties\" /> can be deleted

File: Monitoring.UnitTests/packages.config [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NLog" version="4.0.1" targetFramework="net45" />
-  <package id="PostSharp" version="5.0.44" targetFramework="net45" developmentDependency="true" />
-  <package id="PostSharp.Redist" version="5.0.44" targetFramework="net45" />
+  <package id="PostSharp" version="6.0.28" targetFramework="net45" developmentDependency="true" />
+  <package id="PostSharp.Redist" version="6.0.28" targetFramework="net45" />
```
Comment (linoyisr) [inline Monitoring.UnitTests/packages.config]: This file can be deleted

File: Monitoring.NuGet/Monitoring.NuGet.csproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (linoyisr) [inline Monitoring.NuGet/Monitoring.NuGet.csproj]: Monitoring.NuGet.csproj can be deleted from the repository

File: Monitoring.UnitTests/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -1,41 +1,9 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
-using PostSharp.Extensibility;
+﻿using PostSharp.Extensibility;
```
Comment (linoyisr) [inline Monitoring.UnitTests/Properties/AssemblyInfo.cs]: This file can be deleted

**Comment by linoyisr** [comment]
You need to update also the NuGet.exe and NuGetPack.exe to the version that supported standard (under .NuGetPack)
(You can take the exe files from caching respository)
after this - make sure the nupkg is built properly

File: Monitoring/Monitoring.csproj [hunk 5]
```diff
@@ -1,87 +1,20 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\PostSharp.5.0.44\build\PostSharp.props" Condition="Exists('..\packages\PostSharp.5.0.44\build\PostSharp.props')" />
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (linoyisr) [inline Monitoring/Monitoring.csproj:1]: You need to add here the PostBuildEvent that you accidentally deleted (was in the NuGet project) in order to create the nuspec and nupkg files after build

**Comment by linoyisr** [comment]
Missing VersionHistory file update

File: Monitoring.NuGet/Monitoring.NuGet.csproj (hunk 3 above)
Comment (AvitalOfstein) [reply Monitoring.NuGet/Monitoring.NuGet.csproj]: It is deleted, see Aspects.sln changes

Comment (AvitalOfstein) [reply Monitoring.NuGet/Monitoring.NuGet.csproj]: Deleting it also physically

**Review by linoyisr — APPROVED** [review]

**Pull request merged**